build.sbt

```diff
@@ -423,6 +423,12 @@ val mimaFilterSettings = Seq {
     // Documentation for scala.jdk package
     ProblemFilters.exclude[MissingClassProblem]("scala.jdk.package"),
     ProblemFilters.exclude[MissingClassProblem]("scala.jdk.package$"),
+
+    // Convenience method for copying collections to Arrays (scala/scala#9232)
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.IterableOnce.copyElemsToArray"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.immutable.ArraySeq.copyToArray$default$2"),
```

Comment:
For the record, this is safe because the default cannot be used; instead the single-param overload in `IterableOnceOps` is chosen.

Reply:
I am going to add a note of that to this exclude, since that logic is not immediately obvious to me. I actually didn't notice that one of these wasn't for the `copyElemsToArray` method. oops. I'll separate it slightly in a separate group in the code as well